src/createServer.js

```diff
@@ -1,3 +1,41 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { getReqData } = require('./getReqData');
+const { validateReq } = require('./validateReq');
+const { sendReq } = require('./sendReq');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const { text, caseName } = getReqData(req);
+    const errors = validateReq(text, caseName);
+
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = errors.length ? 400 : 200;
```

Comment:
If you will have 3 or more statusCode, what you will do? Add to the ternary?
Would be better to put this code into the function `sendReq`